### PR TITLE
Part of #18714: Replacing deprecated constants in `confirm-subtitle` folder

### PR DESCRIPTION
**Pull request by rohiiittttt**
## Explanation

- This PR is a part of #18714 

Replaced deprecated design system and storybook constants in the confirm-subtitle
ui/components/app/confirm-subtitle

### Before
image
image

### After
image
image

## Pre-merge author checklist

- [x] I've clearly explained:
  - [x] What problem this PR is solving
  - [x] How this problem was solved
  - [x] How reviewers can test my changes
- [x] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

File: ui/components/app/confirm-subtitle/confirm-subtitle.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { useSelector } from 'react-redux';
 import { SECONDARY } from '../../../helpers/constants/common';
 import {
   Color,
-  FONT_WEIGHT,
+  FontWeight,
```
Comment (georgewrmarshall) [inline ui/components/app/confirm-subtitle/confirm-subtitle.js:8]: We could actually remove this because the font-weight for bodyMd is already normal

**Review by georgewrmarshall — CHANGES_REQUESTED** [review]
Hey @rohiiittttt, thanks for your contribution. I think we can remove fontWeight altogether here

File: ui/components/app/confirm-subtitle/confirm-subtitle.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ const ConfirmSubTitle = ({
     <Text
       as="h5"
       ellipsis
-      fontWeight={FONT_WEIGHT.NORMAL}
+      fontWeight={FontWeight.Normal}
```
Comment (georgewrmarshall) [inline ui/components/app/confirm-subtitle/confirm-subtitle.js:38]: Remove as unneeded
```suggestion
```

**Comment by rohiiittttt** [comment]
@georgewrmarshall i have done the changes please check

**Review by georgewrmarshall — DISMISSED** [review]

**Review by georgewrmarshall — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**